bokeh/settings.py

```diff
@@ -243,6 +243,27 @@ def convert_logging(value):
 
     raise ValueError("Cannot convert {} to log level, valid values are: {}".format(value, ", ".join(_log_levels)))
 
+def convert_validation(value):
+    '''Convert a string to a validation level string
+
+    If a validation level string is passed in, it is returned as-is.
+
+    Args:
+        value (str):
+            A string value to convert to a validation level string
+
+    Returns:
+        string
+
+    Raises:
+        ValueError
+
+    '''
+    if value.lower() == "none" or value.lower() == "errors" or value.lower() == "all":
+        return value.lower()
+
+    raise ValueError("Cannot convert {} to validation level string, valid values are: none, errors, all".format(value))
```

Comment:
There is no need to call `value.lower()` four times, also the list of valid values can be defined and re-used:
```suggestion
    VALID_LEVELS = ["none", "errors", "all"]
    
    value = value.lower()
    if value in VALID_LEVELS:
        return value

    raise ValueError(f"Cannot convert {value} to validation level string, valid values are: {VALID_LEVELS!r}")
```